misc_docs/syntax/extension_embed_raw_top_level.mdx

```diff
@@ -0,0 +1,43 @@
+---
+id: "embed-raw-top-level"
```

Comment:
I'd kinda be in favour calling it `raw-top-level-expression`

```suggestion
id: "raw-top-level-expression"
```

Reply:
Lol, I was unsure about naming for this one.

I left a [comment about naming](https://github.com/rescript-association/rescript-lang.org/pull/444#discussion_r705067193).

Take a look, and make a call. Happy to go with your preference.